app/assets/scripts/utils/field-report-constants.js

```diff
@@ -29,8 +30,7 @@ export const visibility = [
   }
 ];
 
-export const countries = [
-  {value: '-- Country --', label: ''},
+export const countryList = [
```

Comment:
@frozenhelium I'm a bit worried about this breaking other places in the app that use the `countries` list?

Reply:
it won't, the `countries` variable is same as before. I've just separated out the list of countries to `countryList` variable and made `countries` to extend `countryList`

Reply:
Missed that. Looks good :+1: